### ISSUE TITLE
FunctionalTestCompile dependencies not seen in idea editor

### ISSUE THREAD

**Issue by avianey** [GoogleCloudPlatform/gradle-appengine-plugin#171]
I have a GAE gradle project imported in AndroidStudio with following dependencies :
```
dependencies {
    appengineSdk 'com.google.appengine:appengine-java-sdk:1.9.14'
    compile 'com.google.appengine:appengine-endpoints:1.9.14'
    compile 'com.google.appengine:appengine-endpoints-deps:1.9.14'
    compile 'javax.servlet:servlet-api:2.5'
    compile 'com.google.guava:guava:18.0'
    compile 'com.ganyo:gcm-server:1.0.2'
    compile 'com.google.code.gson:gson:2.3.1'
    // functional tests
    functionalTestCompile 'org.testng:testng:6.8.21'
}
```
My tests are executed normaly through gradle :

```
:backend:compileFunctionalTestJava
Download https://repo1.maven.org/maven2/org/testng/testng/6.8.21/testng-6.8.21.pom
Download https://repo1.maven.org/maven2/com/beust/jcommander/1.27/jcommander-1.27.pom
Download https://repo1.maven.org/maven2/org/testng/testng/6.8.21/testng-6.8.21.jar
Download https://repo1.maven.org/maven2/com/beust/jcommander/1.27/jcommander-1.27.jar
:backend:processFunctionalTestResources UP-TO-DATE
:backend:functionalTestClasses
:backend:appengineFunctionalTest

BUILD SUCCESSFUL
```

But idea can't see testng dependencies for functionalTest classes in editor "cannot import `import org.testng.annotations.Test;`"
FunctionalTest classes are located in src/functionalTest/java...

**Comment by avianey** [comment]
It works with an additional testCompile entry...

```
dependencies {
    appengineSdk 'com.google.appengine:appengine-java-sdk:1.9.14'
    compile 'com.google.appengine:appengine-endpoints:1.9.14'
    compile 'com.google.appengine:appengine-endpoints-deps:1.9.14'
    compile 'javax.servlet:servlet-api:2.5'
    compile 'com.google.guava:guava:18.0'
    compile 'com.ganyo:gcm-server:1.0.2'
    compile 'com.google.code.gson:gson:2.3.1'
    // functional tests
    functionalTestCompile 'org.testng:testng:6.8.21'
    testCompile 'org.testng:testng:6.8.21'
}
```

**Comment by loosebazooka** [comment]
I believe this was fixed in #153

**Comment by loosebazooka** [comment]
will go out in 1.9.18
